BUILDER_DEV.md

```diff
@@ -1,221 +1,67 @@
 
-Eventually, we will want to harness the power of the Habitat studio to automate a builder dev environment, but this serves as a stop gap for now.
-
-## Officially Supported Dev Environment
-
-This Dev Environment involves spinning up a virtual machine (most of the core contributors use VMWare, but Virtual Box should work as well) running Ubuntu 17.10 desktop OR Ubuntu 17.10 Server.
-
-### Pre-Reqs
-
-* Ubuntu 17.10 Desktop or Ubuntu 17.10 Server Virtual Machine
-* Access to the "core" origin in [Production Builder](https://bldr.habitat.sh). Ask a core maintainer (or in the [Habitat Slack](http://slack.habitat.sh/)) if you do not have access.
-* Access to the Habitat 1Password account (NOTE - we are working on a solution to make this not required, but it is necessary for the time being)
-* All of the following steps should be run within your Virtual Machine
-* A [Github Personal Access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) with all repo and all user permissions.
-* The [Habitat Builder App](https://github.com/apps/habitat-builder) installed on your Github account.
-* Remove the "secure_path" section from your sudoers file (this is required for the make scripts to run correctly). The following command will comment it out:
-```
-$ sudo sed -i.bak '/secure_path/s/^/#/' /etc/sudoers
-```
-* Some of the sample commands below use the 'httpie' tool. Install it if not present on your system (https://github.com/jkbrzt/httpie).
-
-
-### Builder Setup
-* Update your system
-```
-$ sudo apt-get update
-```
-
-* NOTE - if you are running Ubuntu 17.10 Server, the above command may fail with "E: The repository 'cdrom://Ubuntu-Server 17.10 _Artful Aardvark_ - Release amd64 (20171017.1) artful Release' does not have a Release file."  To fix this, run:
-
-```
-$ sudo sed -i.bak '/deb cdrom/s/^/#/g' /etc/apt/sources.list
-```
-
-* Install git
-```
-$ sudo apt-get install git
-```
-
-* Clone the [Habitat repo](https://github.com/habitat-sh/habitat)
-```
-$ git clone https://github.com/habitat-sh/habitat.git
-```
-
-* Copy habitat-builder-dev.2017-10-02.private-key.pem from 1Password into path/to/habitat/repo/.secrets/builder-github-app.pem (NOTE - this is required as of 12/11/17 - we are working on a solution to make access to the Habitat 1Password vault not required for dev environment setup).
-* Set up some environmental variables (I put these in my .bashrc)
-
-```
-export HAB_AUTH_TOKEN="<your github token>"
-export HAB_BLDR_URL="http://localhost:9636"
-export HAB_ORIGIN="core"
-```
-
-* symlink /src to path/to/your/habitat/repo
-
-```
-$ sudo ln -s path/to/your/habitat/repo /src
-```
+## Pre-Requisites
```

Comment:
I think having the `hab` binary installed and the habitat repo cloned are also prerequisites. Those are documented elsewhere, but mentioning that they need to happen and linking to docs would be nice.

Reply:
`direnv` also appears to be a prerequisite